data/qt-example/shader.frag

```diff
@@ -8,4 +8,5 @@ in vec4 color;
 void main()
 {
     fragColor = color;
+    fragColor = vec4(fragColor.x, 0.0, fragColor.y, 1.0);
```

Comment:
Yes that's the test. Adding this line makes it a bit clearer where one would make a change  that can be reloaded.